ast/compile_test.go

```diff
@@ -7454,3 +7454,119 @@ deny {
 	}
 
 }
+
+func TestCompilerWithRecursiveSchemaAvoidRace(t *testing.T) {
+
+	jsonSchema := `{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://github.com/aquasecurity/defsec/tree/master/pkg/rego/schemas/cloud.json",
```

Comment:
[nit] This 404s. it's probably not relevant, but could we just put something example.com-y in here? (also below, `s/github.com.aquasecurity.defsec/example.com.thingy/`)